src/model_config_utils.h

```diff
@@ -129,6 +144,15 @@ Status LocalizePythonBackendExecutionEnvironmentPath(
     const std::string& model_path, inference::ModelConfig* config,
     std::shared_ptr<LocalizedPath>* localized_model_dir);
 
+/// Set execution environments for Python backend based backend, if the
+/// execution environments is not already specified and the execution
+/// environment file '<backend_libdir>/pb_exec_<runtime>.tar.gz' exists.
+/// \param backend_libdir The backend runtime library directory path.
+/// \param model_config The model configuration.
+/// \return The error status.
+Status SetPythonBackendBasedBackendExecutionEnvironment(
```

Comment:
"python based" over "python backend based" everywhere applicable
```suggestion
Status SetPythonBasedBackendExecutionEnvironment(
```

Reply:
Updated: [Shorten function name](https://github.com/triton-inference-server/core/pull/285/commits/f22a84c452a7cf6eac8d103a393b890e8e62ac60)